src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/JavaScriptTestHelper.cs

```diff
@@ -44,6 +44,40 @@ public static DateTime Now()
         {
             return DateTime.Now;
         }
+        public static int optimizedReached = 0;
+        [JSExport]
+        public static void Optimized0V()
+        {
+            optimizedReached++;
+        }
+        [JSImport("invoke0V", "JavaScriptTestHelper")]
+        public static partial void invoke0V();
+
+        [JSExport]
+        public static void Optimized1V(int a1)
+        {
+            optimizedReached+= a1;
+        }
+        [JSImport("invoke1V", "JavaScriptTestHelper")]
+        public static partial void invoke1V(int a1);
+
+        [JSExport]
+        public static int Optimized1R(int a1)
+        {
+            optimizedReached += a1;
+            return a1 + 1;
+        }
+        [JSImport("invoke1R", "JavaScriptTestHelper")]
+        public static partial int invoke1R(int a1);
+
+        [JSExport]
+        public static int Optimized2R(int a1, int a2)
+        {
+            optimizedReached += a1+ a2;
```

Comment:
If there are specific reasons why the tests have these shapes, it might be good to have a comment about that (other than just "I'm testing functions that happen to have this signature"), like that we broke them somehow in the past. But they look fine otherwise and don't need an explanation